shark/shark_benchmark_runner.py

```diff
@@ -149,8 +149,8 @@ def benchmark_all_csv(
     ):
         self.setup_cl(inputs)
         field_names = [
-            "platform",
             "model",
+            "platform",
```

Comment:
some formatter requires the re-order ?

Reply:
Just changing the column order so that model comes first. This should do the trick